cmd/xpumanager_sidecar/README.md

```diff
@@ -64,11 +65,60 @@ $ kubectl patch daemonsets.apps intel-xpumanager --patch-file 'https://github.co
 
 NOTE: The sidecar patch will remove other resources from the XPU-Manager container. If your XPU-Manager daemonset is using, for example, the smarter device manager resources, those will be removed.
 
-#### Verify Sidecar Functionality
+### Verify Sidecar Functionality
 
 You can verify the sidecar's functionality by checking node's xe-links labels:
 
 ```bash
 $ kubectl get nodes -A -o=jsonpath="{range .items[*]}{.metadata.name},{.metadata.labels.gpu\.intel\.com\/xe-links}{'\n'}{end}"
 master,0.0-1.0_0.1-1.1
 ```
+
+### Use HTTPS with XPU-Manager
+
+XPU-Manager can be configured to use HTTPS on the metrics interface. For the gunicorn sidecar, cert and key files have to be added to the command:
+```
+      - command:
+        - gunicorn
+...
+        - --certfile=/certs/tls.crt
+        - --keyfile=/certs/tls.key
```

Comment:
Add also example of mounting the secret volume(s) for these?

Reply:
I added it in the cert-manager section below.